### PR TITLE
Remove broadcast function

### LINKED ISSUE #6775: Remove fake broadcast operations against builder queues
Currently, we're using a fake broadcast task to trigger removal of paths on all build servers. With the move to scaling groups, the webs will no longer be tracking the individual build servers, and so our broadcast method of looping over `APP_BUILD_SERVERS` will not work anymore.

It's not clear if we even need these operations anymore however. This would probably be the place to start, and if not, we can start stripping out code when we're close to single process builders.

Otherwise, we might need to look into proper broadcast on the build queues.

### PR DESCRIPTION
**Pull request by stsewd**
We don't need this anymore

Closes #6775

### CODE REVIEW

**Comment by stsewd** [comment]
Also, not sure about the remove_dirs function, but I can try removing it where we can in another PR

File: readthedocs/core/utils/general.py
Comment (ericholscher) [inline readthedocs/core/utils/general.py:24]: We might be able to remove the entire `wipe` concept, but that's probably for an additional PR.

Comment (ericholscher) [reply readthedocs/core/utils/general.py:24]: But this logic doesn't actually work -- we definitely don't want to execute it on the webs, do we?

Comment (stsewd) [reply readthedocs/core/utils/general.py:24]: Yeah, I'll look where this can be removed

**Comment by ericholscher** [comment]
@stsewd what is this blocked on?

**Comment by stsewd** [comment]
@ericholscher just want to be sure we remove calls to this function from other repos first (ext and .com)

**Comment by stsewd** [comment]
Only `#311` from ext needs to be merged to unblock this

**Review by humitos — APPROVED** [review]

**Pull request merged**
